quadkey/bq/test/quadkey_integration.js

```diff
@@ -0,0 +1,171 @@
+                    let rows;
+                    await assert.doesNotReject( async () => {
+                        const [job] = await client.createQueryJob({ query: query });
+                        [rows] = await job.getQueryResults();
+                    });
+                    assert.equal(rows.length, 1);
+                    let currentParent = rows[0].currentParent;
+
+                    query = `SELECT \`${BQ_PROJECTID}\`.\`${BQ_DATASET_QUADKEY}\`.PARENT(
+                        \`${BQ_PROJECTID}\`.\`${BQ_DATASET_QUADKEY}\`.ST_ASQUADINT(ST_GEOGPOINT(${lng}, ${lat}),${z})) as parent;`;
+                    rows;
+                    await assert.doesNotReject( async () => {
+                        const [job] = await client.createQueryJob({ query: query });
+                        [rows] = await job.getQueryResults();
+                    });
+                    assert.equal(rows.length, 1);
+                    assert.equal(rows[0].parent, currentParent);
+                }
+            }
+        }
+    });
+
+    it ('Children should work at any level of zoom', async () => {
+        let z, lat, lng, cont;
+        for(z = 5; z < 30; z = z + 20)
+        {
+            for(lat = -90; lat <= 90; lat = lat + 120)
+            {
+                for(lng = -180; lng <= 180; lng = lng + 200)
+                {
+                    let query = `SELECT \`${BQ_PROJECTID}\`.\`${BQ_DATASET_QUADKEY}\`.ST_ASQUADINT(ST_GEOGPOINT(${lng}, ${lat}),${z + 1}) as currentChild;`;
+                    let rows;
+                    await assert.doesNotReject( async () => {
+                        const [job] = await client.createQueryJob({ query: query });
+                        [rows] = await job.getQueryResults();
+                    });
+                    assert.equal(rows.length, 1);
+                    let currentChild = rows[0].currentChild;
+
+                    query = `SELECT \`${BQ_PROJECTID}\`.\`${BQ_DATASET_QUADKEY}\`.CHILDREN(
+                        \`${BQ_PROJECTID}\`.\`${BQ_DATASET_QUADKEY}\`.ST_ASQUADINT(ST_GEOGPOINT(${lng}, ${lat}),${z})) as children;`;
+                    rows;
+                    await assert.doesNotReject( async () => {
+                        const [job] = await client.createQueryJob({ query: query });
+                        [rows] = await job.getQueryResults();
+                    });
+                    assert.equal(rows.length, 1);
+                    assert.equal(rows[0].children.length, 4);
+                    let childs = rows[0].children;
+                    cont = 0;
+                    childs.forEach((element) => {
+                        if(currentChild === element)
+                        {
+                            ++cont;
+                        }
+                    });
+                    assert.equal(cont,1);
+                }
+            }
+        }
```

Comment:
Where are the tests for all the other functions (`ST_ASQUADINTPOLYFILL`, `ST_GEOGFROMQUADINT_BOUNDARY`, `KRING`...)? Is there something missing in this PR?